src/components/article/ListPagination.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Pagination, Grid } from 'semantic-ui-react';
+import '../../styles/Articles.scss';
+
+
+class ListPagination extends Component {
+    static currentLocalStorage = (localStorage) => {
+        let currentPage = localStorage.getItem('page');
+        if (currentPage !== null) {
+            return currentPage;
+        }
+    }
+    onPageChange = (event, data) => {
+        const { fetchArticles } = this.props;
+        const url = 'https://ah-backend-summer-staging.herokuapp.com/api/v1/articles';
+        const limit = (count, p) => `limit=${count}&offset=${p ? p * count : 0}`;
+        const page = data.activePage;
+        const pagination = `?${limit(10, page - 1)}`;
```

Comment:
`10` and `1` are magic numbers. It is hard to know what they represent.
The same issue is on line 25.